src/Bundle/DependencyInjection/SyliusGridExtension.php

```diff
@@ -77,4 +75,97 @@ public function getConfiguration(array $config, ContainerBuilder $container): Co
 
         return $configuration;
     }
+
+    private function loadPersistence(array $drivers, array $grids, LoaderInterface $loader, ContainerBuilder $container): void
+    {
+        $availableDrivers = $this->getAvailableDrivers($container);
+
+        // Enable all available drivers if there is no configured drivers
+        $drivers = [] !== $drivers ? $drivers : $availableDrivers;
+
+        $gridDrivers = $this->getGridDrivers($grids);
+
+        $this->checkConfiguredDrivers($drivers, $availableDrivers, $gridDrivers);
+
+        foreach ($drivers as $enabledDriver) {
+            if ($enabledDriver === SyliusGridBundle::DRIVER_DOCTRINE_PHPCR_ODM) {
+                @trigger_error(sprintf(
+                    'The "%s" driver is deprecated in Sylius 1.3. Doctrine PHPCR will no longer be supported in Sylius 2.0.',
+                    SyliusGridBundle::DRIVER_DOCTRINE_PHPCR_ODM,
+                ), \E_USER_DEPRECATED);
+            }
+
+            $loader->load(sprintf('services/integrations/%s.xml', $enabledDriver));
+        }
+    }
+
+    /**
+     * @param array<string, array{driver?: array{name: string}}> $drivers
+     *
+     * @return array<string, string>
+     */
+    private function getGridDrivers(array $drivers): array
```

Comment:
`GridDriver` sounds like something else than regular `Driver` from $availableDrivers, which is as far as I understand is not true.

```suggestion
    private function resolveDriversUsedInGrids(array $grids): array
```